SU-SWS/saa_victoria_subtheme
# ADAPT-456: Started check mark paragraph styles.

# READY FOR REVIEW

# Summary
- Add check mark paragraph style

# Review By (Date)
- When does this need to be reviewed by?

# Urgency
- How critical is this PR?

# Steps to Test

1. Pull down the branch
2. Clear the site cache
3. Add this to a text area and compare to figma:  https://www.figma.com/file/TXNBECAGcEkZ3vwc6xcZ3g/SAA-Design-System?node-id=33%3A0

```
<h2>Check Mark Section</h2>

<p class="check-mark-paragraph">Set up your <a href="https://alumni.stanford.edu/get/page/register" title="https://alumni.stanford.edu/get/page/register"><strong>Stanford Alumni account</strong></a>. It’s the gateway to all things awesome.</p>

<p class="check-mark-paragraph">Don’t miss out on alumni events and resources! Be sure to log in and <a href="https://alumni.stanford.edu/get/page/my-account/profile/" title="https://alumni.stanford.edu/get/page/my-account/profile/"><strong>provide an email address</strong></a> you check often.</p>

<p class="check-mark-paragraph">Speaking of email addresses...get your free lifetime <a href="https://alumni.stanford.edu/get/page/saa/resources/alumniaccounts" title="https://alumni.stanford.edu/get/page/saa/resources/alumniaccounts"><strong>Stanford alumni email address</strong></a>! (Psst! @alumni.stanford.edu looks great on resumes, LinkedIn, business cards, and more.) Make sure it forwards to an email you check often.</p>

<p class="check-mark-paragraph">Don’t miss out on opportunities to connect with local alumni and invitations to regional events. <a href="https://alumni.stanford.edu/get/page/my-account/profile/" title="https://alumni.stanford.edu/get/page/my-account/profile/"><strong>Update your mailing address</strong></a> now!</p>

<p class="check-mark-paragraph">Give us a follow! We’re @stanfordalumni on <a href="https://www.instagram.com/stanfordalumni" title="https://www.instagram.com/stanfordalumni"><strong>Instagram</strong></a>, <a href="https://twitter.com/StanfordAlumni" title="https://twitter.com/StanfordAlumni"><strong>Twitter</strong></a> and <a href="https://www.facebook.com/stanfordalumni" title="https://www.facebook.com/stanfordalumni"><strong>Facebook</strong></a>.</p>

<p class="check-mark-paragraph">Explore Stanford <a href="https://alumni.stanford.edu/get/page/events" title="https://alumni.stanford.edu/get/page/events"><strong>alumni events</strong></a> and <a href="https://alumni.stanford.edu/get/page/groups/browse/interest" title="https://alumni.stanford.edu/get/page/groups/browse/interest"><strong>groups</strong></a>.</p>

<p class="check-mark-paragraph">Did you know there is a LinkedIn group for Stanford alums only? Join <a href="https://www.linkedin.com/groups/74186/" title="https://www.linkedin.com/groups/74186/"><strong>Stanford Alumni on LinkedIn</strong></a> today!</p>

<p class="check-mark-paragraph">Don’t miss out on your great <a href="https://alumni.stanford.edu/get/page/membership/newgradmembers" title="https://alumni.stanford.edu/get/page/membership/newgradmembers"><strong>SAA member benefits</strong></a>!</p>

<p class="check-mark-paragraph">Check out the <a href="https://alumni.stanford.edu/get/page/life-long-learning/learn-join" title="https://alumni.stanford.edu/get/page/life-long-learning/learn-join"><strong>SAA email subscriptions</strong></a> (STANFORD magazine, engaging offerings from Alumni Education and much more).</p>

```

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket ADAPT-456
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
